Clean up networking/virtual-ips.md (#50237)
kubernetes/website: content/en/docs/reference/networking/virtual-ips.md

A backend is chosen (either based on session affinity or randomly) and packets are
redirected to the backend without rewriting the client IP address.

> This same basic flow executes when traffic comes in through a NodePort or

Reviewer: People might think they can `kubectl get NodePort`, so I recommend:
```suggestion
This same basic flow executes when traffic comes in through a `type: NodePort` Service, or
```

`nftables` mode should be aware that some features work slightly
differently the `nftables` mode:

- **NodePort interfaces**: In `iptables` mode, by default,
  [NodePort services](/docs/concepts/services-networking/service/#type-nodeport)
  are reachable on all local IP addresses. This is usually not what
  users want, so the `nftables` mode defaults to

>   `--nodeport-addresses primary`, meaning NodePort services are only

Reviewer: ```suggestion
  `--nodeport-addresses primary`, meaning Services using `type: NodePort` are only
```

  reachable on the node's primary IPv4 and/or IPv6 addresses. You can
  override this by specifying an explicit value for that option:
  e.g., `--nodeport-addresses 0.0.0.0/0` to listen on all (local)
  IPv4 IPs.

> - **NodePort services on `127.0.0.1`**: In `iptables` mode, if the

Reviewer: ```suggestion
- `type: NodePort` **Services on `127.0.0.1`**: In `iptables` mode, if the
```

Aside: is this behavior IPv4-only?

  `--nodeport-addresses` range includes `127.0.0.1` (and the option
  `--iptables-localhost-nodeports false` option is not passed), then

>   NodePort services are reachable even on "localhost" (`127.0.0.1`).

Reviewer: ```suggestion
  Services of `type: NodePort` are reachable even on "localhost" (`127.0.0.1`).
```

  In `nftables` mode (and `ipvs` mode), this will not work. If you
  are not sure if you are depending on this functionality, you can
  check kube-proxy's
  `iptables_localhost_nodeports_accepted_packets_total` metric; if it

>   is non-0, that means that some client has connected to a NodePort
>   service via `127.0.0.1`.

Reviewer: ```suggestion
  is non-0, that means that some client has connected to a `type: NodePort`
  Service via localhost / loopback.
```

  proportionally distributed across zones. To mitigate this, consider the
  following strategies:

  * [Pod Topology Spread Constraints](/docs/concepts/scheduling-eviction/topology-spread-constraints/):
    Use Pod Topology Spread Constraints to distribute your pods more evenly
    across zones.

  * Zone-specific Deployments: If you expect to see skewed traffic patterns,
    create a separate Deployment for each zone. This approach allows the
    separate workloads to scale independently. There are also workload
    management addons available from the ecosystem, outside the Kubernetes
    project itself, that can help here.

> * **Implementation-specific behavior:** Each dataplane implementation may handle

Reviewer: ```suggestion
* **Implementation-specific behavior:** Each data plane implementation may handle
```
